.github/workflows/docs-checks.yml

```diff
@@ -158,6 +158,12 @@ jobs:
         run: |
           make html --debug --jobs $(nproc) SPHINXOPTS="-W --keep-going"
 
+      - name: Check External Links in Sphinx Documentation (Optional)
+        run: |
+          cd docs
+          make linkcheck
```

Comment:
```suggestion
        working-directory: ./docs/source-${{ matrix.pkg-name }}
        run: make linkcheck
```

Reply:
Updated with this change.

Reply:
Btw, you can easily just accept a proposal =)